### PR TITLE
To-Do-List : List Structure

### PR DESCRIPTION
**Pull request by Zohra-Neda**
**In this projects we :** 

1. Create src folder
2. Create index.js file
3. Create template.html file
4. create style.css file

### CODE REVIEW

**Review by julie-ify — COMMENTED** [review]
Hi @Zohra-Neda,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Your readme file does not have a key features section to match the link in the menu. Also ensure you remove the live demo section if you have no demo link

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    isCompleted: true,
+  },
+  {
+    description: 'task2',
+    isCompleted: false,
+  },
+  {
+    description: 'task3',
+    isCompleted: false,
+  },
```
Comment (julie-ify) [inline src/index.js]: - [ ] This should be `completed`, not `isCompleted`.

- [ ] You are missing the `index` key also.

  Each task should be like this:

  ```
  {
    description: 'task 1',
    completed: true,
    index: 1
  }
  ```

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+todos.forEach((todo, index) => {
+  content += `
+    <li id="todo-item">
+      <div id=${index}>
+        <span class="bi-circle"></span>
+        <p>${todo.description}</p>
+        <p>${todo.isCompleted}</p>
+      </div>
+    </li>`;
+});
```
Comment (julie-ify) [inline src/index.js]: - [ ] These should be check boxes not list item identifiers. Kindly fix that.

  image

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Zohra-Neda,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Zohra-Neda👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**